hack/build/ci/preflight.sh

```diff
@@ -20,6 +20,7 @@ check_image() {
     1> "${PREFLIGHT_REPORT_NAME}" 2> "${PREFLIGHT_LOG}"
   echo "${PREFLIGHT_EXECUTABLE} returned ${?}"
   cat "${PREFLIGHT_LOG}"
+  rm -rf artifacts preflight.log
```

Comment:
```suggestion
  rm -rf artifacts "${PREFLIGHT_LOG}"
```

can we move this into a `cleanup()` function so the intention is clearer?

Reply:
This line changed to only `rm -rf artifacts` which should be reasonable and does not require an own function.